Source/Core/Core/CoreTiming.cpp

```diff
@@ -26,6 +26,8 @@
 #include "VideoCommon/VideoBackendBase.h"
 #include "VideoCommon/VideoConfig.h"
 
+#include "TimePlayed.h"
+
```

Comment:
Is this an intentional change? The file is otherwise unchanged, and could just be removed from the commit.